app/utils/key-camelizer.js

```diff
@@ -0,0 +1,56 @@
+'use strict'
+
+function convertToCamelCase (objectKey) {
+  const camelizedString = objectKey.replace(/[-_\s]+(.)?/g, (match, chr) => chr ? chr.toUpperCase() : '')
+  return camelizedString.substr(0, 1).toLowerCase() + camelizedString.substr(1)
+}
+
+function separateWords (objectKey) {
+  const options = {}
```

Comment:
`options` is always an empty object?

Reply:
Think that was copied/pasted from the humps module. As it passed my tests I think I just neglected to look at it properly. I've refactored and renamed to `convertToSnakeCase`. As you mentioned `options` wasn't really doing anything so it's been deleted. However, and I've passed the `separator` and `split` in as arguments in case it needs to be used in isolation as a separate module, but perhaps that means that the name will no longer be appropriate?